<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportImageDialog</class>
 <widget class="QDialog" name="ExportImageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>619</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Export Images</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Image Format</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QRadioButton" name="pnmRadio">
        <property name="text">
         <string>PNM</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QRadioButton" name="pngRadio">
        <property name="text">
         <string>PNG</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="tiffRadio">
        <property name="text">
         <string>TIFF</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="bmpRadio">
        <property name="text">
         <string>BMP</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="dicomRadio">
        <property name="text">
         <string>DICOM</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="jpegRadio">
        <property name="text">
         <string>JPEG</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="jpegGroup">
     <property name="title">
      <string>JPEG Quality</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
      <item>
       <widget class="QSlider" name="jpegQualitySlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>20</number>
        </property>
        <property name="value">
         <number>90</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Worst</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Best</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="bmpGroup">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>BMP Bit Options</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="bmp8BitRadio">
        <property name="text">
         <string>8 Bits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="bmp24BitRadio">
        <property name="text">
         <string>24 Bits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="bmp32BitRadio">
        <property name="text">
         <string>32 Bits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="bmpDefaultRadio">
        <property name="text">
         <string>Default</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="pnmGroup">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>PNM Bit Options</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="pnm8BitRadio">
        <property name="text">
         <string>8 Bits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pnm16BitRadio">
        <property name="text">
         <string>16 Bits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pnmPastelRadio">
        <property name="text">
         <string>Pastel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pnmDefaultRadio">
        <property name="text">
         <string>Default</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Export Destination</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="exportDestEdit">
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>0</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectFolderButton">
        <property name="minimumSize">
         <size>
          <width>107</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>107</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Select Folder</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0,0,0">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textDirection">
        <enum>QProgressBar::TopToBottom</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startBtn">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>ExportImageDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>586</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
